server/src/main/java/org/elasticsearch/common/time/DateFormatters.java

```diff
@@ -85,13 +85,13 @@
         .optionalStart()
         .appendFraction(MILLI_OF_SECOND, 3, 3, true)
         .optionalEnd()
+        .optionalEnd()
```

Comment:
this is just moving the zoneId below the optional section of seconds/milliseconds. Hard to see with this formatting